service/src/main/java/bio/terra/workspace/service/workspace/model/AwsCloudContext.java

```diff
@@ -1,72 +1,69 @@
 package bio.terra.workspace.service.workspace.model;
 
-import bio.terra.common.exception.SerializationException;
+import bio.terra.aws.resource.discovery.Environment;
 import bio.terra.workspace.common.exception.InternalLogicException;
 import bio.terra.workspace.common.exception.StaleConfigurationException;
-import bio.terra.workspace.db.DbSerDes;
 import bio.terra.workspace.db.model.DbCloudContext;
 import bio.terra.workspace.generated.model.ApiAwsContext;
-import bio.terra.workspace.service.workspace.exceptions.InvalidSerializedVersionException;
+import bio.terra.workspace.service.workspace.exceptions.InvalidCloudContextStateException;
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Objects;
 import javax.annotation.Nullable;
 
 public class AwsCloudContext implements CloudContext {
-  private final String majorVersion;
-  private final String organizationId;
-  private final String accountId;
-  private final String tenantAlias;
-  private final String environmentAlias;
-  private final @Nullable CloudContextCommonFields commonFields;
+  private final @Nullable AwsCloudContextFields contextFields;
+  private final CloudContextCommonFields commonFields;
 
   @JsonCreator
   public AwsCloudContext(
-      @JsonProperty("majorVersion") String majorVersion,
-      @JsonProperty("organizationId") String organizationId,
-      @JsonProperty("accountId") String accountId,
-      @JsonProperty("tenantAlias") String tenantAlias,
-      @JsonProperty("environmentAlias") String environmentAlias,
+      @JsonProperty("contextFields") @Nullable AwsCloudContextFields contextFields,
       @JsonProperty("commonFields") CloudContextCommonFields commonFields) {
-    this.majorVersion = majorVersion;
-    this.organizationId = organizationId;
-    this.accountId = accountId;
-    this.tenantAlias = tenantAlias;
-    this.environmentAlias = environmentAlias;
+    this.contextFields = contextFields;
```

Comment:
Just to confirm: the AwsCloudContext objects were previously serialized in flights but never in the DB?
I think the change to fields here will cause issues with any in-progress flights, but given the infrequency and how fast those flights are, I think that should be safe enough

Reply:
I think they are serialized in the DB. It should be OK. The serialized form is now made and deserialized in `AwsCloudContextFields` and should be identical with what used to be in `AwsCloudContext`. That is true of all of the context objects.

Reply:
Did the serialization store any class information? That would cause problems if we did, but if it's just by fields we should be fine

Reply:
The database serializer does not store class info.